.github/workflows/ci.yml

```diff
@@ -30,6 +30,9 @@ jobs:
             ${{ runner.os }}-go-
       - name: run "go generate ./..."
         run: go generate ./...
+      - name: run "go generate internal/typedoc"
+        working-directory: internal/typedoc 
+        run: go generate ./...
```

Comment:
I stumbled upon this by mistake but it appeared to generate a change which wasn't generated by default with `go generate ./...` (see failing CI run [here](https://github.com/ariga/atlas/runs/5046361816?check_suite_focus=true)). I am happy to remove this if it's not necessary